### PR TITLE
[ABO-202] Whitelist Freemint Transfer for accounts

### PR DESCRIPTION
**Pull request by DidacSF**
## Description

See details in https://ajunanetwork.atlassian.net/browse/ABO-202

## Type of changes

- [ ] `build`: Changes that affect the build system or external dependencies (eg, Cargo, Docker)
- [ ] `ci`: Changes to CI configuration
- [ ] `docs`: Changes to documentation only
- [x] `feat`: Changes to add a new feature
- [ ] `fix`: Changes to fix a bug
- [ ] `refactor`: Changes that do not alter functionality
- [ ] `style`: Changes to format the code
- [ ] `test`: Changes to add missing tests or correct existing tests

## Checklist

- [ ] Tests for the changes have been added
- [ ] Necessary documentation is added (if appropriate)
- [ ] Formatted with `cargo fmt --all`
- [ ] Linted with `cargo clippy --all-features --all-targets`
- [ ] Tested with `cargo test --workspace --all-features --all-targets`

### CODE REVIEW

File: pallets/ajuna-awesome-avatars/src/lib.rs [hunk 1]
```diff
@@ -148,6 +155,28 @@ pub mod pallet {
...
+		fn get() -> bool {
+			true
+		}
+	}
+
+	/// Flog to mark free mints as transferable by all accounts or not,
+	/// if free mint transfer is blocked, only whitelisted accounts
+	/// can still make transfers
+	#[pallet::storage]
+	pub type MintTransferAllowed<T: Config> =
```
Comment (darkfriend77) [inline pallets/ajuna-awesome-avatars/src/lib.rs]: This should be handled in the `GlobalConfig` having one flag for general FreeMintTransfers and one for WhiteListed FreeMintTransfers

https://github.com/ajuna-network/Ajuna/blob/develop/pallets/ajuna-awesome-avatars/src/types/config.rs#L101

Add a FreeMintConfig

**Review by darkfriend77 — COMMENTED** [review]
I think it is important to have the configuration consistently in the GlobalConfig, for the turn off and on, whitelist storage for accounts is fine

File: pallets/ajuna-awesome-avatars/src/lib.rs (hunk 1 above)
Comment (DidacSF) [reply pallets/ajuna-awesome-avatars/src/lib.rs]: Okay, updated PR!

**Review by darkfriend77 — CHANGES_REQUESTED** [review]
There is an inconsistency here.

each config has an open flag which should open or close this function,

so if we do this properly the freemint should have it's own config, and transfer should only apply to avatars.

I know this will involve a storage migration.

**Review by darkfriend77 — APPROVED** [review]

**Pull request merged**